File: code_llm/multiagentes/gpt-4o-mini/Cubo Preto.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;

    int none = (N > 2) ? (N - 2) * (N - 2) * (N - 2) : 0;
    int one = (N > 2) ? 6 * (N - 2) * (N - 2) : 0;
    int two = (N > 2) ? 12 * (N - 2) : 0;
    int three = 8;

    cout << none << endl;
    cout << one << endl;
    cout << two << endl;
    cout << three << endl;

    return 0;
}

File: output/cbsoft_sbes_2026/gpt_5_4/zero/python_img/Cubo Preto.py
n = int(input().strip())

zero = max(0, (n - 2) ** 3)
one = 6 * max(0, (n - 2) ** 2)
two = 12 * max(0, n - 2)
three = 8

print(zero)
print(one)
print(two)
print(three)